python/setup.py

```diff
@@ -202,6 +206,7 @@ def initialize_options(self):
         '_dataset_orc',
         '_dataset_parquet',
         '_acero',
+        '_azurefs',
```

Comment:
Similarly, move this below near the other filesystem options?

Reply:
Again, its here because of an attempt to order alphabetically but obviously there are plenty of exceptions to that so I will move it as you suggest.